syntax = "proto3";
package plugin;





/*
CreatePlugin请求
*/
message CreatePluginRequest {
    /*
    名称
    */
    string name = 1 ;
    /*
    备注
    */
    string memo = 2 ;
    /*
    部署路径
    */
    repeated string deployPath = 3 ;
}

/*
CreatePlugin返回
*/
message CreatePluginResponse {
    /*
    ID
    */
    string id = 1 ;
}

/*
CreatePluginApi返回
*/
message CreatePluginResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    plugin.CreatePluginResponse data = 4 ;
}

